packages/demo-html/public/behavioral-html/exit-html.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Open: exit (via embed code)</title>
+    <style>
+      #visual-threshold {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 50px;
+        background: rgba(255, 0, 0, 0.2);
+        pointer-events: none;
+      }
+    </style>
+    <link rel="stylesheet" href="/lib/css/popup.css" />
```

Comment:
what about other demos?

Reply:
I am fixing it now, I am running the tests to see if all the demos work fine